src/parser_utils.js

```diff
@@ -14,17 +14,38 @@ function childrenByName(node, name) {
 }
 
 /**
- * Parses a node text (for legacy support).
- * @param  {Object} node - The node to parse the text from.
+ * Parses a node value
+ * @param  {Object} node - The node to parse the value from.
  * @return {String}
  */
-function parseNodeText(node) {
+function parseNodeValue(node) {
+  const childNodes = node && node.childNodes && [...node.childNodes];
+
+  // Trying to find and parse cdata as JSON
+  const cdatas =
+    childNodes &&
+    childNodes.filter(childNode => childNode.nodeName === '#cdata-section');
+  if (cdatas && cdatas.length > 0) {
+    try {
+      return JSON.parse(cdatas[0].data);
+    } catch (e) {}
```

Comment:
Should we at least log this?

Reply:
Plus according to mdn:
"The CDATASection interface represents a CDATA section that can be used within XML to include extended portions of unescaped text. The symbols < and & don’t need escaping as they normally do when inside a CDATA section."

So some vmaps may use it to pass some content other than an Object, so if `JSON.parse` fails shouldn't we return that content?